tokio/src/fs/write.rs

```diff
@@ -6,6 +6,12 @@ use std::{io, path::Path};
 /// contents of `contents` to it.
 ///
 /// This is the async equivalent of [`std::fs::write`][std].
+///
+/// This operation is implemented by running the equivalent blocking operation
+/// on a separate thread pool using [`spawn_blocking`].
+///
+/// [`spawn_blocking`]: crate::task::spawn_blocking
+
 ///
 /// [std]: fn@std::fs::write
```

Comment:
Some extra empty lines here.
```suggestion
/// [`spawn_blocking`]: crate::task::spawn_blocking
/// [std]: fn@std::fs::write
```